Cargo.toml

```diff
@@ -20,7 +20,7 @@ members = [
 # Tokio ecosystem
 tokio = { version = "1.37.0", features = ["full", "tracing"] }
 tokio-stream = "0.1.11"
-tokio-util = { version = "0.7.10", features = ["codec", "compat"] }
+tokio-util = { version = "0.7.13", features = ["codec", "compat"] }
```

Comment:
Do we actually need this patch version?

Reply:
yes - it introduces a function called `run_until_cancelled`, which we use currently in one place, but the usage can be extended to other places. this PR is already destructive as it is, but `run_until_cancelled` can very easily replace this construct:
```rust
tokio::select! {
  _ = cancellation_token.cancelled() => {}
  _ = other_async_action_like_a_long_sleep => {}
}
```

Reply:
Maybe 0.7 will do the trick as well then? Instead of juggling patch versions all the time